lib/runner/cli/argv-setup.js

```diff
@@ -495,6 +495,10 @@ module.exports = new (function () {
         description: 'Set the udid of real iOS device'
       });
 
+      this.option('rerun-failed', {
```

Comment:
why `--retry`?

Reply:
Because `rerun-failed` seems better objective centric. While using only `retry` may not fulfil it. But still open for suggestions.

Reply:
Also `--retry` is already being used by the existing session-based retry functionality.

Reply:
Ah, I that case `rerun-failed` sounds good.